SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -200,6 +200,11 @@ bool USpatialActorChannel::ReplicateActor()
 	check(!Closing);
 	check(Connection);
 	check(Connection->PackageMap);
+
+	if (Actor->Role != ROLE_Authority)
```

Comment:
Should we move this into IsReadyForReplication? Requires moving the Actor check also.